submissions/karelskiy/tiny-JS-world/index.js

```diff
@@ -0,0 +1,36 @@
+const population = [
+    {
+        species: 'dog',
+        name: 'Toby',
+        gender: 'male',
+        legs: 4,
+        hands: 0,
+        saying: 'woof-woof!'
+    },
+    {
+        species: 'cat',
+        name: 'Mrs. Velasquez',
+        gender: 'female',
+        legs: 4,
+        hands: 0,
+        saying: 'meow-meow!'
+    },
+    {
+        species: 'human',
+        name: 'Enrico',
+        gender: 'male',
+        legs: 2,
+        hands: 2,
+        saying: 'Holla'
+    },
+    {
+        species: 'human',
+        name: 'Angel',
+        gender: 'female',
+        legs: 2,
+        hands: 2,
+        saying: 'Ehi, buongiorno'
+    }
+];
+
+population.forEach(el => print(Object.values(el).join('; ')));
```

Comment:
Good job.
But imagine your peer adds an inhabitant with a different order of props. E.g.
```
{
  name: 'Becky',
  species: 'catwoman',
  gender: 'female',
  hands: 2,
  legs: 2,
  saying: 'Meow!',
}
```
Your overall output will be somewhat confusing.

Moreover in real life there is no guarantee that order of props in object internal presentation would be the same as in the source code. Another use case is composition. That is when you combine two or more objects into one with `Object.assign()`. You may want props be listed in a different order.

So it is good to manage the order explicitly.

Consider the following fragment:
```
const obj = {
  someProp: 5,
  anotherProp: 6,
  yetAnotherProp: 7,
};

['anotherProp', 'yetAnotherProp', 'someProp'].map(prop => obj[prop]); // result: [6, 7, 5]
```

You will find this technique useful in one of the following tasks.

Reply:
As i understand, i need to display value of properties and also describe their in a few words?

Reply:
Also i have little problem on GitHub, but it will be resolve soon.

Reply:
> As i understand, i need to display value of properties and also describe their in a few words?

Not necessarily. This is the way to guarantee the order of properties for iteration/output.